README.md

```diff
@@ -137,21 +113,39 @@ Please see the full disclaimer [below](#disclaimer).
 ### Legend
 
 - 🔹 Important or often overlooked tip
-- ❗ “Serious” gotcha where risks or costs are significant
-- 🔸 A gotcha or limitation to be aware of
+- ❗ Serious danger or pitfall where risks or costs are significant
+- 🔸 A warning or limitation to be aware of
 - 🌪 Controversial topic where informed opinion varies significantly
 - ☝️ Common confusion or misunderstanding, such as confusing terminology
 - 📥 PDF or form or download
 - 🚧 Areas where improvement or expansion is needed
 
-### Please help improve this Guide!
+### Please help!
 
 This Guide is a living publication.
 If you have an idea or contribution that might improve this Guide, please add suggestions
 in the margins.
 We gladly credit all contributors.
 
-## Companies, Compensation, and Stock
+## Roadmap
+
+🚧 Overview and infographic of concept dependencies and how sections fit together.
+
+## Fundamentals of Stock Corporations
+
+When it comes to equity compensation, everything depends on understanding some fundamental
+questions:
+
+- What is stock, and how do I own it?
+  What are shares?
+- Is stock the same as equity?
+- What is my equity worth?
+- If I have or will get stock at some point, can I sell it?
```

Comment:
Rather, "If I have or will get stock at some point, what can I do with it?" ("What does it mean to have equity?")